tests/snappi/qos/files/packet_reorder_helper.py

```diff
@@ -0,0 +1,321 @@
+import time
+import logging
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.snappi.snappi_helpers import get_dut_port_id
+from tests.common.snappi.common_helpers import disable_packet_aging
+from tests.common.snappi.port import select_ports, select_tx_port
+from tests.common.snappi.snappi_helpers import wait_for_arp
+
+logger = logging.getLogger(__name__)
+
+FLOW_NAME = 'IP-IP Test Flow'
+TX_PORT_NAME = None
+RX_PORT_NAME = None
+FLOW_AGGR_RATE_PERCENT = 70
+LARGER_PKT_SIZE = 512
+SMALLER_PKT_SIZE = 256
+PKT_STEP_SIZE = LARGER_PKT_SIZE - SMALLER_PKT_SIZE
+UDP_SRC_PORT = 63
+UDP_DST_PORT = 63
+UDP_PKT_LEN = 190
+TOTAL_NUM_PKTS = 100000
+EXP_FLOW_DUR_SEC = 3
+PKT_SEND_DELAY = 2
```

Comment:
Why we need PFC for this test?

Reply:
The packets sent are lossless but no PFC pause frames are being sent

Reply:
> The packets sent are lossless but no PFC pause frames are being sent

Then remove this variable.